cli/cmd/plugin/ui/web/tanzu-ui/src/App.tsx

```diff
@@ -16,7 +16,7 @@ import UnmanagedClusterLanding from './views/unmanaged-cluster/UnmanagedClusterL
 import VSphere from './components/VSphere';
 import Welcome from './views/welcome/Welcome';
 import WorkloadClusterWorkflow from './views/workload-cluster/WorkloadClusterWorkflow';
-import DockerManagementClusterSimple from './views/management-cluster/DockerManagementClusterSimple';
```

Comment:
Likewise `Simple` vs `Advanced` and perhaps `Basic` instead?

Reply:
Agreed, basic sounds better